components/main-nav.vue

```diff
@@ -25,12 +29,14 @@ export default {
       default: null
     }
   },
-  computed: {
-    navItems() {
-      return this.mainNavItems.filter((navItem) => {
-        const [{ text }] = navItem.primary.title
-        return text !== 'Accueil'
-      })
+  methods: {
+    isPixProActive(index) {
+      if (index !== this.mainNavItems.length - 3) return false
+      if (
+        ['/employeurs', '/mediation-numerique'].includes(this.$nuxt.$route.path)
+      )
+        return true
+      return false
     }
```

Comment:
pourquoi pas juste `return ['/employeurs', '/mediation-numerique'].includes(this.$nuxt.$route.path)` ?

Reply:
J'y avais pas pensé 👍